src/django_components/expression.py

```diff
@@ -48,8 +50,7 @@ def __init__(self, parser: Parser, expr_str: str) -> None:
         # Copy the Parser, and pass through the tags and filters available
         # in the current context. Thus, if user calls `{% load %}` inside
         # the expression, it won't spill outside.
-        lexer = Lexer(self.expr)
-        tokens = lexer.tokenize()
+        tokens = parse_template(self.expr)
```

Comment:
This ensures that, when we use a template tag inside a component input like so:

```django
{% component
    description="{% lorem 3 w %}"
%}
```

then the inner `{% lorem 3 w %}` will be parsed by the updated parser too.